Week-5/assignment-1/index.js

```diff
@@ -0,0 +1,53 @@
+import fetch from "node-fetch";
+
+//completing the tasks with Generators
+function doTask1() {
+  let result = fetch("https://jsonplaceholder.typicode.com/users");
+  result
+    .then((val) => {
+      return val.json();
+    })
+    .then((val) => {
+      console.log(val);
+      me.next();
+    });
+}
+
+function doTask2() {
+  setTimeout(() => {
+    console.log("completed task 2");
+    me.next();
```

Comment:
not a good practice to access global var in func. Pass it through printResults generator

Reply:
not sure how can i implement this lets discus on 1-1